nautobot/docs/installation/nautobot.md

```diff
@@ -0,0 +1,255 @@
+- Run the database schema migrations
+- Aggregate static resource files on disk
+
+!!! important
+    PostgreSQL and Redis must have been successfully installed before continuing with deployment steps. If you haven't done that yet, please visit the guide on [Installing Nautobot](../../installation/#install-nautobot).
+
+## Create the Nautobot System User
+
+Create a system user account named `nautobot`. This user will own all of the Nautobot files, and the Nautobot web services will be configured to run under this account. 
+
+```no-highlight
+$ sudo useradd --create-home --system --shell /bin/bash nautobot
+```
+
+## Upgrade Pip
+
+[Pip]((https://pip.pypa.io/)) is Python's package installer and is referred interchangeably as `pip` or `pip3`. For the purpose of this document, we'll deliberately be referring to it as `pip3`.
+
+Many common issues can be solved by running the latest version of Pip. Before continuing with installing Nautobot, upgrade Pip to its latest release:
+
+```no-highlight
+$ sudo pip3 install --upgrade pip
+```
+
+## Choose your `NAUTOBOT_ROOT`
+
+This is where everything related to Nautobot will be installed. We're going to use this value across the documentation. You'll need to set the `NAUTOBOT_ROOT` environment variable to tell Nautobot where to find its files and settings at `/opt/nautobot`.
+
+```no-highlight
+$ export NAUTOBOT_ROOT=/opt/nautobot
+```
+
+## Create the Virtual Environment
+
+A Python [virtual environment](https://docs.python.org/3/tutorial/venv.html) or *virtualenv* is like a container for a set of Python packages. A virtualenv allows you to build environments suited to specific projects without interfering with system packages or other projects. When installed per the documentation, Nautobot uses a virtual environment in production.
+
+We're going to create the virtualenv as our `NAUTOBOT_ROOT` as the root user to bootstraps the `/opt/nautobot` directory and populate it with a self-contained Python environment. 
+
+```no-highlight
+$ sudo python3 -m venv $NAUTOBOT_ROOT
+```
+
+Next, change ownership of `NAUTOBOT_ROOT` to the `nautobot` user:
+
+```no-highlight
+$ sudo chown -R nautobot:nautobot $NAUTOBOT_ROOT
+```
+
+## Sudo to nautobot
+
+Now that we've created the virtualenv, the remaining steps will be performed as the `nautobot` user. 
+
+!!! warning
+    Don't skip this step!!
+
+    It's important to install Nautobot as the `nautobot` user so that we don't have to worry about fixing permissions later.
+
+```no-highlight
+$ sudo su - nautobot
+```
+
+After becoming `nautobot`, we need to set the `NAUTOBOT_ROOT` environment variable again for this user (since it is a fresh session as far as the system is concerned):
+
+```no-highlight
+$ export NAUTOBOT_ROOT=/opt/nautobot
+```
+
+### Add `NAUTOBOT_ROOT` to `.bashrc`
+
+For bonus points, add this to `~/.bashrc` for `nautobot`:
+
+```no-highlight
+$ echo "export NAUTOBOT_ROOT=/opt/nautobot" >> ~/.bashrc
```

Comment:
`echo "export NAUTOBOT_ROOT=$NAUTOBOT_ROOT" >> ~/.bashrc`

Reply:
I feel like that would be confusing to some since it's not explicit.

Reply:
Fair, I was trying to allow users to change to /some/random/path early on and not breaking things.